Building.md

```diff
@@ -0,0 +1,43 @@
+This is the general getting started guide for FullScreenShenanigans.
+
+### Table of Contents
+
+1. [Obtaining the Source Code](#obtaining-the-source-code)
+2. [Build Process](#build-process)
+3. [Using the Console](#using-the-console)
+
+## Obtaining the Source Code
+
+In order to play FullScreenShenanigans, you first need to obtain the source code. There are two ways to do this:
+
+A. Clone the repository:
```

Comment:
These steps are unclear. If you mention something like as Git Bash and don't provide any context, that's confusing.

You can replace steps 1 and 2 with installing using this link: http://git-scm.com/book/en/v2/Getting-Started-Installing-Git